nexus/db-queries/src/db/datastore/volume.rs

```diff
@@ -1583,6 +1626,274 @@ impl DataStore {
     }
 }
 
+pub struct VolumeReplacementParams {
+    pub volume_id: Uuid,
+    pub region_id: Uuid,
+    pub region_addr: SocketAddrV6,
+}
+
+impl DataStore {
+    /// Replace a read-write region in a Volume with a new region.
+    pub async fn volume_replace_region(
+        &self,
+        target: VolumeReplacementParams,
```

Comment:
I'm not attached, but don't like `original` in this case. What about `target_for_replacement`?

Reply:
It's the overuse of "target" that confuses me, because we have a field in the
VCR that is named target.  We are (at some point in the this process) 
replacing the VCR target field with something new.

So, I'm requesting the only time we say "target" is when we are talking about
the field in the VCR, and use another word or set of words as names of things
elsewhere in this process.

Reply:
changed to `existing` in d3596e0b6